omod/src/main/webapp/resources/htmlforms/HMIS-TB-003-TBClientCard-SummaryPage.xml

```diff
@@ -0,0 +1,650 @@
+	                                        </div>
+	                                        <div class="col-md-4">
+	                                        	<label>DRTB No.</label>
+	                                            <obs conceptId="165843" />
+	                                        </div>
+	                                    </div>
+	                                </div>
+	                            </div>
+	                            <div class="card">
+	                                <div class="card-header">
+	                                    TB/HIV
+	                                </div>
+	                                <div class="card-body">
+	                                    <div class="row">
+	                                        <div class="col-md-4">
+	                                            <label>HIV Status</label>
+	                                            <obs conceptId="165396" answerConceptIds="90167,165397,165350,90001"
+	                                                 answerLabels="Negative, Newly Tested Positive, Known Positive, Unknown" 
+	                                                 class="long-value-select" id="165396"/>
+	                                        </div>
+	                                        <div class="col-md-4">
+	                                        	<label>Date</label>
+	                                            <obs conceptId="164431" id="hiv-test-date"/>
+	                                        </div>
+	                                    </div>
+	                                   	<div class="row">
+	                                        <div class="col-md-4">
+	                                            <label>ART Status</label>
+	                                            <obs conceptId="1358" answerConceptIds="1362,1367" 
+	                                            	 answerLabels="Initiated on ART, Already on ART" 
+	                                            	 class="long-value-select" id="1358"/>
+	                                        </div>
+	                                        <div class="col-md-4">
+	                                            <label>Date</label>
+	                                            <obs conceptId="99161" id="99161" />
+	                                        </div>
+	                                        <div class="col-md-4">
+	                                        	<label>ART No.</label>
+	                                            <obs conceptId="99431" id="99431" />
+	                                        </div>
+	                                    </div>
+	                                    <div class="row">
+	                                        <div class="col-md-4">
+	                                            <label>CPT/Dapson</label>
+	                                            <obs conceptId="90171" answerConceptIds="99022,1193" 
```

Comment:
That concept is causing this error
https://gist.github.com/slubwama/f18cb270ad9d6a042c53b883c0a89581

Reply:
I have changed it to 92